CoreGraphNode/src/au/gov/asd/tac/constellation/graph/node/create/NewSchemaGraphAction.java

```diff
@@ -68,6 +68,7 @@ public class NewSchemaGraphAction extends AbstractAction implements DynamicMenuC
     private static Map<String, String> templates;
     private static JMenu menu;
     private static final Map<String, Icon> ICON_CACHE = new HashMap<>();
+    public static final String FORWARD_SLASH = "/";
```

Comment:
I'm not sure it makes sense to have this constant sitting in this class. Maybe it would fit better in `SeparatorConstants`?